roles/system/files/50unattended-upgrades

```diff
@@ -104,7 +104,7 @@ Unattended-Upgrade::Automatic-Reboot-Time "05:00";
 
 // Use apt bandwidth limit feature, this example limits the download
 // speed to 70kb/sec
-//Acquire::http::Dl-Limit "70";
+Acquire::http::Dl-Limit "250";
```

Comment:
Please drop this, just let it download as fast as it can. This will not help relieve any theoretical lag.

Reply:
This was one of the suggestions to determine what was causing the lag a few weeks, but I already changed it back on the servers, so I will fix it here.